### ISSUE TITLE
Build Action: BundleResource does not exist.

### ISSUE THREAD

**Issue by ohumidk** [dotnet/maui#7938]
In xamarin forms you are able to change the build action in an iOS project to BundleResource.
I use this for settings up firestore with my xamarin.forms app.

I am rewriting my app in MAUI but I am unable to set my file's build action to BundleResource.
Was this left off intentionally?

**Comment by xtuzy** [comment]
maybe latest VisualStudio don't support, you can try specify it at .csproj

**Comment by mattleibow** [comment]
I am not sure about this issue. OP is asking about BundleResource, this is available in the IDE but it may be better to make use of the Resources/Raw folder as that translates to the same thing.

@xtuzy is also correct, you can always manually set that file in the csproj.

@mmendoza-apa can you open a new issue with the errors you are facing?

**Comment by DanielMuhlig** [comment]
Same/similar issue in Visual Studio Version 17.5.3

1. Copied GoogleService-Info.plist to \Platforms\iOS
2. BundleResource missing as build action.
3. Manually set it in .csproj
<ItemGroup Condition="'$(TargetFramework)' == 'n:et7.0-ios'">
	<BundleResource Include="Platforms\iOS\GoogleService-Info.plist" />
</ItemGroup>
4. The GoogleService-Info.plist disappeared from \Platform\iOS in Solution Explorer

**Comment by ohumidk** [comment]
My update on this: it was a problem and I really like firebase so I rewrote my project and switched to the Flutter framework. Works a lot better than this. 

Maui is still premature.

**Comment by mattleibow** [comment]
I think all you need to do I put the json file in a Resources folder in the platform, so /Platforms/iOS/Resources.

This will automatically get the correct build action. Most items are now default by folder.

**Comment by DanielMuhlig** [comment]
@ohumidk Not very helpful. 

I am as seasoned C# developer (20+ years) and love the MS stack and are just starting on mobile development. 

It's like if I was asking how to change a lightbulp on a Mercedes and your comment would be that Mercedes is a bad car and that I wouldn't have those problems if I bought a Toyota instead.

I can't (won't) switch technology stack just because there is a bump on the road!

**Comment by ohumidk** [comment]
@DanielMuhlig, I love the MS stack as well. I've been a developer for 2+ years. I would still say I'm fresh in the game. 

Where I work, we use the MS stack. We are also about to release our rewrite of our mobile app in .NET MAUI and it just works a lot better when using other .NET products. 

I personally think it's okay to switch. I barely have skin in the game. There are so many tech stacks out there in the world. I want to expose myself to other products. Flutter is pretty nice and it works great with firebase. MAUI is great too, but it has the same issues that exist in XF plus other complications. There may be workarounds now but I personally couldn't get it to work for myself at the time i wrote the issue. I was not to far into writing my personal project so I decided it switch and I'm glad I did.
